arviz/plots/backends/bokeh/ppcplot.py

```diff
@@ -260,18 +284,34 @@ def plot_ppc(
                     line_alpha=alpha,
                 )
                 glyph.level = "overlay"
+                legend_it.append((label, [glyph]))
 
+            all_scatter = []
             for vals, y in zip(pp_sampled_vals, y_rows[1:]):
                 vals = np.ravel(vals)
                 yvals = np.full_like(vals, y, dtype=np.float64)
                 if jitter:
                     yvals += np.random.uniform(low=scale_low, high=scale_high, size=len(vals))
-                ax_i.scatter(vals, yvals, fill_color=color, size=markersize, fill_alpha=alpha)
+                scatter = ax_i.scatter(
+                    vals, yvals, fill_color=color, size=markersize, fill_alpha=alpha
+                )
+                all_scatter.append(scatter)
 
+            legend_it.append(("{} predictive".format(group.capitalize()), all_scatter))
             ax_i.yaxis.major_tick_line_color = None
             ax_i.yaxis.minor_tick_line_color = None
             ax_i.yaxis.major_label_text_font_size = "0pt"
 
+        if legend:
+            legend = Legend(
+                items=legend_it,
+                location="top_left",
+                orientation="vertical",
+            )
+            ax_i.add_layout(legend)
+            if textsize is not None:
+                ax_i.legend.label_text_font_size = str(textsize) + "pt"
```

Comment:
I would use an f-string here instead of adding strings

Reply:
@OriolAbril  Done!